packages/test/test-utils/src/containerUtils.ts

```diff
@@ -0,0 +1,59 @@
+/*!
```

Comment:
nit: Maybe this file should be called connectionUtils.ts.

Reply:
I can see both approaches. I thought of it as utility functions related to the container (so that would get a container as parameter), and right now the only thing we have are functions related to the container's connection. Once we have more utilities one way or the other we can keep the filename or change it.